tools/tesseratos/src/tesseratos/toolbox/plugin.cpp

```diff
@@ -0,0 +1,47 @@
+#include <imgui.h>
+
+
+#include <cubos/engine/imgui/plugin.hpp>
+
+#include <tesseratos/toolbox/plugin.hpp>
+
+using cubos::core::ecs::EventWriter;
+using cubos::core::ecs::Read;
+using cubos::core::ecs::Write;
+
+
+using cubos::engine::Cubos;
+using cubos::core::ecs::Write;
+
+using namespace tesseratos;
+
+static void showToolbox(Write<Toolbox> toolbox)
+{
+    ImGui::Begin("Toolbox");
+
+    for(auto a : toolbox->toolsMap)
```

Comment:
Kinda confused here. You can make it more readable by using modern C++ stuff: `for (auto& [key, val] : map)`